docker/benchmarking/init_geth.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+PW="1234567"
+ACCTS=30
+
+go get -d github.com/ethereum/go-ethereum
+go install github.com/ethereum/go-ethereum/cmd/geth
+
+geth --identity "geth-benchmark" init --datadir data bench-geth-genesis.json
+
+# Generate 30 accounts
+echo -e 'Generating Accounts\n'
+for i in $(seq 1 $ACCTS)
+do
+  geth --datadir "data" account new --password <(echo $PW)
+  # i dont know what the windows command would be to get the last created account thats what this does in windows
+  for /f "delims=" %%i in ('geth --datadir "data" account list') do set account=%%i
+  ./benchmarking ag -ac %account% -am 10000000
+  # echo "Generated test$i account"
```

Comment:
I just kinda guessed on these commands to hope they work on mac. Bash scripts suck no matter what on windows so some of these will need fixed.